src/calculator/pump/PumpData.cpp

```diff
@@ -0,0 +1,414 @@
+                   std::string driveType, std::string flangeConnectionClass, std::string flangeConnectionSize,
+                   int numShafts, int speed, int numStages,  int yearlyOperatingHours, int yearInstalled,
+                   int finalMotorRpm, double inletDiameter, double weight, double outletDiameter,
+                   double percentageOfSchedule, double dailyPumpCapacity, double measuredPumpCapacity,
+                   double pumpPerformance, double staticSuctionHead, double staticDischargeHead, double fluidDensity,
+                   double lengthOfDischargePipe, double pipeDesignFrictionLosses, double maxWorkingPressure,
+                   double maxAmbientTemperature, double maxSuctionLift, double displacement, double startingTorque,
+                   double ratedSpeed, double shaftDiameter, double impellerDiameter, double efficiency,
+                   double output60Hz, double minFlowSize, double pumpSize, bool outOfService)
+        : manufacturer(std::move(manufacturer)), model(std::move(model)), type(std::move(type)), serialNumber(std::move(serialNumber)),
+          status(std::move(status)),  pumpType(std::move(pumpType)), radialBearingType(std::move(radialBearingType)),
+          thrustBearingType(std::move(thrustBearingType)), shaftOrientation(std::move(shaftOrientation)), shaftSealType(std::move(shaftSealType)),
+          fluidType(std::move(fluidType)), priority(std::move(priority)), driveType(std::move(driveType)), flangeConnectionClass(std::move(flangeConnectionClass)),
+          flangeConnectionSize(std::move(flangeConnectionSize)), numShafts(numShafts), speed(speed), numStages(numStages),
+          yearlyOperatingHours(yearlyOperatingHours), yearInstalled(yearInstalled), finalMotorRpm(finalMotorRpm),
+          inletDiameter(inletDiameter), weight(weight), outletDiameter(outletDiameter), percentageOfSchedule(percentageOfSchedule), dailyPumpCapacity(dailyPumpCapacity),
+          measuredPumpCapacity(measuredPumpCapacity), pumpPerformance(pumpPerformance), staticSuctionHead(staticSuctionHead),
+          staticDischargeHead(staticDischargeHead), fluidDensity(fluidDensity), lengthOfDischargePipe(lengthOfDischargePipe),
+          pipeDesignFrictionLosses(pipeDesignFrictionLosses),  maxWorkingPressure(maxWorkingPressure),
+          maxAmbientTemperature(maxAmbientTemperature), maxSuctionLift(maxSuctionLift),  displacement(displacement),
+          startingTorque(startingTorque), ratedSpeed(ratedSpeed), shaftDiameter(shaftDiameter), impellerDiameter(impellerDiameter),
+          efficiency(efficiency), output60Hz(output60Hz), minFlowSize(minFlowSize), pumpSize(pumpSize), outOfService(outOfService)
+{}
+
+
+const bool &PumpData::getOutOfService() const {
+    return outOfService;
+}
+
+void PumpData::setOutOfService(const bool &outOfService)  {
+    PumpData::outOfService = outOfService;
```

Comment:
In the other parts of the code, a [trailing underscore is used for the member variable](https://github.com/ORNL-AMO/AMO-Tools-Suite/blob/f03a9fb6dec4c479d97a36d8965ed33531d566ef/include/calculator/furnace/FlowCalculationsEnergyUse.h#L347) instead of using the same name as the input parameter. That said, this works too. I just don't know if you want consistency throughout the library.

Reply:
nope

Reply:
@pshires It was a cursory glance, but i'm glad I found the one example that is inconsistent.